src/nixdimensions.cc

```diff
@@ -40,4 +40,45 @@ namespace nixdimensions {
 
         return sb.array();
     }
+
+    void sampled_position_at(const extractor &input, infusor &output) {
+        nix::SampledDimension dim = input.entity<nix::SampledDimension>(1);
+        const mwSize index = static_cast<mwSize>(input.num<double>(2));
```

Comment:
I know it is just a quick fix, but `mwSize` should be `size_t` because that is what `dim.positionAt(index)` expects.